src/app/shared/components/alert/alert.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="alert alert--dark">
-  <svg-icon src="assets/images/icon-sprite.svg#error" class="icon alert__icon"></svg-icon>
+  <svg-icon [attr.src]="'assets/images/icon-sprite.svg#'+alertType" class="icon alert__icon"></svg-icon>
```

Comment:
This made the icon dynamic regarding [this code sandbox](https://codesandbox.io/s/happy-chatelet-zsl93)